docs/assets-update.md

```diff
@@ -0,0 +1,67 @@
+# Clear Containers assets updates 
+
+Clear Containers make use of additional resources required to create a Virutual
+Machine Contianer some of this compoents are the Guest OS
+[rootfs and kernel](https://github.com/clearcontainers/runtime/blob/master/docs/architecture/architecture.md#assets).
+The `rootfs` guest OS asset is based packages and produced by Clear Linux\*. This
+docuement describe how and when Clear Containers Guest assets will be updated.
+
+## Clear Containers Kernel
+
+[Clear Containers kernel](https://github.com/clearcontainers/linux). It is a
+Linux\* kernel based in the latest LTS vanilla version from kernel.org. This vanilla LTS
+kernel is patched with some [kernel patches](https://github.com/clearlinux-pkgs/linux-container/)
+need for Clear Containers. A LTS version is used to reduce
+the number of updates we would have to roll out and use a more "stable" and tested
+kernel. 
+
+### Clear Containers Kernel update
+
+Clear Containers release is produced weekly. Each time a new kernel version is out
+should be updated updated in 
+[Clear Containers packaging repository](https://github.com/clearcontainers/packaging)
+and [Clear Containers Linux kernel](https://github.com/clearcontainers/linux)
+repository. The latest versions in these repositories will be used for
+the release.
+
+### Clear Containers Image.
+
+The Clear Containers image guest OS system is generated using Clear Linux packages. This image 
+Contain a minimal system for Clear Containers. The image is generated multiple times a day as
+part of the Clear Linux Release Cycle. 
+
+### Update Clear Containers image.
+
+The Clear Containers image should be updated only when critical updates were
+done to the packages used by the Clear Containers guest OS:
+
+- Systemd
+- [Clear Containers Agent](https://github.com/clearcontainers/agent)
+- iptables
+- core-utils
+
+This will be verified each release using the script
+https://github.com/clearcontainers/packaging/blob/master/scripts/get-image-changes.sh
+
+The image must be updated in [Clear Containers packaging repository](https://github.com/clearcontainers/packaging)
+and defined in https://github.com/clearcontainers/runtime/blob/master/versions.txt
+in the [Clear Containers Release](https://github.com/clearcontainers/runtime/wiki/Release-Process).
+
+## Clear Containers Continuous integration.
+
+Official Clear Containers Packages are hosted in OBS build system. In order to
+avoid availability issues the kernel and image for Clear Containers continuous 
+integration system used is downloaded from Clear Linux\* packages:
+
+```
+https://download.clearlinux.org/releases/${CLEAR_VERSION}/clear/x86_64/os/Packages/linux-container-${KERNEL_VERSION}.rpm 
+```
+
+The Clear Containers assets packages provide identica kernel and images
+for the various Linux Distributions.
+
+### Security
+
+In case of any critical update is done in the kernel or Clear Contaienrs image
```

Comment:
In the case of critical updates to the kernel or the Clear Containers image, a new Clear Containers release will be rolled out immediately.